src/main/java/teammates/common/util/Const.java

```diff
@@ -1028,6 +1028,11 @@ private Const() {
         public static final String PENDING_COMMENT_CLEARED_EMAIL_WORKER_URL =
                 "/worker/pendingCommentClearedEmail";
 
+        public static final String SEARCHABLE_DOCUMENTS_PRODUCTION_FOR_COMMENTS_QUEUE_NAME =
```

Comment:
It seems the other variables are named in the imperative mood. Also, searchable is a bit verbose. Maybe something like PRODUCE_SEARCH_DOCUMENTS_COMMENTS_QUEUE_NAME?